src/tests/ftest/deployment/critical_integration.py

```diff
@@ -77,9 +77,8 @@ def test_passwdlessssh_versioncheck(self):
         result_client_server = daos_server_version_list[0][1:] == dmg_version_list[0]
 
         # libfabric version check
-        all_nodes = self.hostlist_servers + self.hostlist_clients
-        all_nodes = list(dict.fromkeys(all_nodes))
-        list_all_nodes = ",".join(all_nodes)
+        all_nodes = self.hostlist_servers | self.hostlist_clients
+        list_all_nodes = str(all_nodes)
```

Comment:
FYI, the `|` operand is not supported for lists. So until rebased, this will fail